promoter/database.go

```diff
@@ -0,0 +1,132 @@
+package promoter
+
+import (
+	"context"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"go.sia.tech/siad/crypto"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/mongo/readconcern"
+	"go.mongodb.org/mongo-driver/mongo/readpref"
+	"go.mongodb.org/mongo-driver/mongo/writeconcern"
+)
+
+const (
+	dbName                  = "siacoinpromoter"
+	colWatchedAddressesName = "watchedaddresses"
```

Comment:
Our current convention is snake_case for collections and kebab-case for databases.